src/main/java/com/google/neighborgood/servlets/EditTaskServlet.java

```diff
@@ -124,6 +124,22 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
 
+    // Get the task overview from the form input
+    String taskOverview = "";
+
+    input = request.getParameter("task-overview-input");
+    // If the input is valid, set the taskOverview value to the input value
+    if (input != null) {
+      taskOverview = input.trim();
+    }
+
+    // If input task overview is empty, reject the request to edit and send a 400 error.
+    if (taskOverview.equals("")) {
+      System.err.println("The input task overview is empty");
+      response.sendRedirect("/400.html");
```

Comment:
let's open an issue to track removal of the 400 redirects for these errors

Reply:
Sure. I will open an issue right now.